api/types/usertasks/descriptions/ec2-ssm-script-failure.md

```diff
@@ -0,0 +1,3 @@
+Teleport was able to reach the SSM Agent inside the EC2 instance, however the script returned an error.
+
+You can click below in the Invocation URL and get further details on why the script failed.
```

Comment:
The URL seems to be missing?

Reply:
The URL will appear below.
We have the the description and then a table with affected instances
That table will contain, for each instance, its invocation URL.